components/studentPortal/freemium/modals/Exit.tsx

```diff
@@ -0,0 +1,21 @@
+import router from "next/router";
+import React from "react";
+import { Button } from "../../../ui/Button";
+
+const handlePlansSignUp = () => {
+  router.push("/plans");
+};
+
+export default function Exit() {
```

Comment:
I'd probably name this component ExitModal because it could be confusing when you call "Exit" inside code